kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/food2door/ProducerExtraFoodShop.java

```diff
@@ -0,0 +1,15 @@
+package com.kodilla.good.patterns.challenges.food2door;
+
+public class ProducerExtraFoodShop extends Producer {
+    public ProducerExtraFoodShop(String producerId, String name, String location) {
+        super(producerId, name, location);
+    }
+
+    @Override
+    protected boolean process(OrderRequest orderRequest) {
+        if (orderRequest.getSupplier().getHeadquarters().equals(Headquarters.CHELMNO)) {
```

Comment:
return orderRequest.getSupplier().getHeadquarters().equals(Headquarters.CHELMNO);

Wystarczy

Reply:
Do tego Chelmno powinno byc w parametrze, bo teraz metoda to nie process, tylko "sprawdzCzyToChemlno"